packages.yml

```diff
@@ -25,3 +25,9 @@ packages:
 
   - package: fivetran/twitter_ads
     version: [">=0.6.0", "<0.7.0"]
+
+  # - package: fivetran/amazon_ads
+  #   version: [">=0.1.0", "<0.2.0"]
+  - git: https://github.com/fivetran/dbt_amazon_ads.git
+    revision: main
+    warn-unpinned: false
```

Comment:
Friendly reminder to switch before release 😄

Reply:
done